packages/react/src/components/tabs/tabs.tsx

```diff
@@ -286,10 +313,14 @@ export const Tabs: VoidFunctionComponent<Props> = ({
                         </TabButton>
                     ))}
                     {onAddTab && (
-                        <AddButton buttonType='tertiary' type="button" onClick={onAddTab}>
-                            <AddIcon name="plusSign" size='16' aria-hidden="true" focusable={false} />
-                            {t('addTab')}
-                        </AddButton>
+                        <>
+                            {addButton.tooltipContent && (
+                                <Tooltip label={addButton.tooltipContent} desktopPlacement="top">
+                                    {addButtonComponent}
+                                </Tooltip>
+                            )}
+                            {!addButton.tooltipContent && (addButtonComponent)}
+                        </>
```

Comment:
Peut-être pas besoin du fragment
```suggestion
                        addButton.tooltipContent ? (
                            <Tooltip label={addButton.tooltipContent} desktopPlacement="top">
                                {addButtonComponent}
                            </Tooltip>
                        ) : addButtonComponent
```